java/elemental2/chrome/integer_entities.txt

```diff
@@ -0,0 +1,7 @@
+elemental2.chrome.MessageSender.frameId
+elemental2.chrome.Tab.id
+elemental2.chrome.Tab.index
+elemental2.chrome.Tab.windowId
+elemental2.chrome.Tab.openerTabId
+elemental2.chrome.Tab.width
+elemental2.chrome.Tab.height
```

Comment:
these are all the simple integer-seeming types i could find in the [`chrome` extern namespace](https://github.com/google/closure-compiler/blob/master/contrib/externs/chrome.js), at first glance